src/main/java/net/java/otr4j/session/Session.java

```diff
@@ -648,12 +648,29 @@ private String handleDataMessage(DataMessage data) throws OtrException {
                 if (tlvs != null && tlvs.size() > 0) {
                     for (TLV tlv : tlvs) {
                         switch (tlv.getType()) {
-                            case TLV.DISCONNECTED:
+                            // TVL.PADDING // TLV0
```

Comment:
I think we should make this case explicit. So TLV.PADDING: break; That way, if ever we decide to have a "default" case for unsupported types, we can be sure that we *only* catch the unsupported types.